comp/core/gui/guiimpl/gui.go

```diff
@@ -124,14 +124,14 @@ func newGui(deps dependencies) provides {
 	sessionExpiration := deps.Config.GetDuration("GUI_session_expiration")
 	g.auth = newAuthenticator(authToken, sessionExpiration)
 
-	router.HandleFunc("/auth", g.getAccessToken).Methods("GET")
-	// Serve the (secured) index page on the default endpoint
-	securedRouter := router.PathPrefix("/").Subrouter()
-	securedRouter.HandleFunc("/", generateIndex).Methods("GET")
-
-	// Mount our (secured) filesystem at the view/{path} route
-	securedRouter.PathPrefix("/view/").Handler(http.StripPrefix("/view/", http.HandlerFunc(serveAssets)))
+	// register the public routes
+	publicRouter.HandleFunc("/", renderIndexPage).Methods("GET")
+	publicRouter.HandleFunc("/auth", g.getAccessToken).Methods("GET")
+	// Mount our filesystem at the view/{path} route
+	publicRouter.PathPrefix("/view/").Handler(http.StripPrefix("/view/", http.HandlerFunc(serveAssets)))
```

Comment:
`/view` used to be on the authenticated handler but now it's public, is that on purpose ?

Reply:
`/view` is only static files that are used to render the frontend to the user.
The GUI work as a Single Page Application, we ship the layout and then the authentication is made via XHR/AJAX server-side on every requests, so no need to reduce access of views IMO 
Having it public also allow anyone that access to the GUI URL to get information on how to connect to it.